amt/locale/nl_NL/LC_MESSAGES/messages.po

```diff
@@ -527,28 +527,58 @@ msgstr "Groeperen op"
 msgid "Select group by"
 msgstr "Selecteer groepering"
 
+#: amt/site/templates/projects/details_base.html.j2:19
+msgid "Delete algoritmic system"
+msgstr "Verwijder Algoritmesysteem"
+
 #: amt/site/templates/projects/details_base.html.j2:26
+msgid "Are you sure you want to delete your algoritmic system "
+msgstr "Weet u zeker dat u uw algoritmische systeem wilt verwijderen?"
```

Comment:
I think this is not being rendered completely right: see here
<img width="657" alt="Screenshot 2024-11-11 at 14 26 02" src="https://github.com/user-attachments/assets/49aa5000-422f-4eac-b1c9-2da2613177ca">

I think we want 1 question mark, after the name, not one before and after the name